src/serialize.rs

```diff
@@ -134,6 +134,12 @@ pub fn node_from_stream<T: Allocator>(allocator: &T, f: &mut dyn Read) -> std::i
                     ops.push(ParseOp::Cons);
                     ops.push(ParseOp::SExp);
                     ops.push(ParseOp::SExp);
+                } else if b[0] == 0x00 {
+                    values.push(allocator.null());
+                } else if b[0] == 0x01 {
+                    values.push(allocator.one());
+                } else if b[0] == 0x80 {
```

Comment:
This is a lone zero, which is NOT a null

Reply:
Sorry, `0x80` is an empty string... it's `0x00` that a lone zero (which is not a null). I think.

Reply:
fixed